src/fiatconnect/ReviewScreen.test.tsx

```diff
@@ -102,9 +108,57 @@ describe('ReviewScreen', () => {
       ])
     })
 
+    it('shows expired dialog when quote is expired', async () => {
+      const expireMs = -100
+      const mockProps = getProps(CICOFlow.CashOut, false, CryptoType.cUSD, expireMs)
+      const quote = _.cloneDeep(mockFiatConnectQuotes[1]) as FiatConnectQuoteSuccess
+      quote.quote.guaranteedUntil = new Date(Date.now() + expireMs).toISOString()
+      const store = createMockStore({
+        fiatConnect: {
+          quotes: [quote],
+        },
+      })
+      const { getByTestId, queryByTestId } = render(
+        <Provider store={store}>
+          <FiatConnectReviewScreen {...mockProps} />
+        </Provider>
+      )
+
+      expect(queryByTestId('expiredQuoteDialog')?.props.visible).toEqual(true)
```

Comment:
```suggestion
      expect(queryByTestId('expiredQuoteDialog')).toBeTruthy()
```

Reply:
similar feedback for all the ones below. Truthy is visible and falsy should be not visible

Reply:
Wouldn't `expect(queryByTestId('expiredQuoteDialog'))` always be truthy because the element always renders? I only change the isVisible prop.

Reply:
Ah I see. I made an incorrect assumption on how you were implementing it. sounds good